gateway/http.d/apicast.conf.liquid

```diff
@@ -1,5 +1,11 @@
 log_format time '[$time_local] $host:$server_port $remote_addr:$remote_port "$request" $status $body_bytes_sent ($request_time) $post_action_impact';
 
+map $status $extended_access_log {
```

Comment:
Why is this map needed? Sorry, I forgot :sweat_smile:

Reply:
if not nignx will not start due to extended_access_log is not defined.

Reply:
And a simple `set` does not work?